src/types.ts

```diff
@@ -228,7 +283,7 @@ export type ISelectionAction = {
   /** Icon component */
   icon: any;
   /** Action to perform */
-  callback: (selection: Map<number, IPhoto>) => Promise<void>;
+  callback: (selection: any) => Promise<void>;
```

Comment:
The type should be `SelectionProxy` but this doesn't work. Can someone help out here?